qiskit_experiments/library/characterization/correlated_readout_error.py

```diff
@@ -77,11 +78,11 @@ class CorrelatedReadoutError(BaseExperiment):
         .. ref_arxiv:: 1 2006.14044
     """
 
+    @deprecate_arguments({"qubits": "physical_qubits"})
```

Comment:
The downside of using the Qiskit deprecator is that it doesn't say which release will be the last one before the deprecated argument is removed, which we generally do. I'd prefer it if we provide this info to users for such a big API change.

Reply:
Done in 3d09f739